pxr/usd/usdLux/schema.usda

```diff
@@ -187,18 +249,140 @@ class "LightAPI" (
     bool inputs:normalize = false (
         displayGroup = "Advanced"
         displayName = "Normalize Power"
-        doc = """Normalizes power by the surface area of the light.
-        This makes it easier to independently adjust the power and shape
-        of the light, by causing the power to not vary with the area or
-        angular size of the light."""
+        doc = """Normalizes the emission such that the power of the light
+        remains constant while altering the size of the light, by dividing the
+        luminance by the world-space surface area of the light.
+
+        This makes it easier to independently adjust the brightness and size
+        of the light, by causing the total illumination provided by a light to
+        not vary with the area or angular size of the light.
+
+        Mathematically, this means that the luminance of the light will be
+        divided by a factor representing the "size" of the light:
+
+        <center><b>
+                        L<sub>Scalar</sub> = L<sub>Scalar</sub> / sizeFactor
+        </b></center>
+
+        ...where `sizeFactor` = 1 if `normalize` is off, and is calculated
+        depending on the family of the light as described below if `normalize`
+        is on.
+
+        ### DomeLight / PortalLight:
+
+        For a dome light (and its henchman, the PortalLight), this attribute is
+        ignored:
+
+        <center><b>
+                        sizeFactor<sub>dome</sub> = 1
+        </b></center>
+
+        ### Area Lights:
+
+        For an area light, the `sizeFactor` is the surface area (in world
+        space) of the shape of the light, including any scaling applied to the
+        light by its transform stack. This includes the boundable light types
+        which have a calculable surface area:
+
+        - MeshLightAPI
+        - DiskLight
+        - RectLight
+        - SphereLight
+        - CylinderLight
+        - (deprecated) GeometryLight
+
+        <center><b>
+                        sizeFactor<sub>area</sub> = worldSpaceSurfaceArea(light)
+        </b></center>
+
+        ### DistantLight:
+
+        For distant lights, we first define 𝛳<sub>max</sub> as:
+
+        <center><b>
+                𝛳<sub>max</sub> = clamp(toRadians(distantLightAngle) / 2, 0, 𝜋)
+        </b></center>
+
+        Then we use the following formula:
+
+        * <i>if 𝛳<sub>max</sub> = 0:</i>
+        <center><b>
+                sizeFactor<sub>distant</sub> = 1
+        </b></center>
+
+        * <i>if 0 < 𝛳<sub>max</sub> ≤ 𝜋 / 2:</i>
+        <center><b>
+            sizeFactor<sub>distant</sub> = sin²𝛳<sub>max</sub> ⋅ 𝜋
+        </b></center>
+
+        * <i>if 𝜋 / 2 < 𝛳<sub>max</sub> ≤ 𝜋:</i>
+        <center><b>
+                    sizeFactor<sub>distant</sub> =
+                        (2 - sin²𝛳<sub>max</sub>) ⋅ 𝜋
+        </b></center>
```

Comment:
Curious if you have a link to the derivation of this?

If I compute a normalization factor as a fraction of solid angle coverage I get a different answer: https://www.desmos.com/calculator/coa7luomx3

Reply:
It's the integral over _projected_ solid angle. See for example page 11 here: https://www.cs.cornell.edu/courses/cs5625/2022sp/notes/radiometry-notes.pdf

One way of looking at this is it expresses the light in units of irradiance, which is useful. A simpler way to look at it is "the light doesn't get dimmer when you make the angle bigger".

Reply:
Thanks for the explanation @anderslanglands, maybe this should be in the documentation for future reference?

Just a few follow up questions if I may. This normalization is defined for surfaces, does this mean a volume lit with a normalized distant light will not stay the same brightness for different sizes? Should there be a different normalization technique for volumes?

Reply:
Heh, at one point I wrote this explanation up, before axing it because it was too wordy:

https://github.com/pmolodo/OpenUSD/blob/pr/usd-lux-update-doc-normalizingDistantLights.pdf/pxr/usd/usdLux/doxygen/normalizingDistantLights.pdf

(There's also a [.md version](https://github.com/pmolodo/OpenUSD/blob/pr/usd-lux-update-doc-normalizingDistantLights.pdf/pxr/usd/usdLux/doxygen/normalizingDistantLights.md), but it's formatted for Doxygen's markdown flavor, not github's, so not all formating works 100%.)